Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:3.9.9-buster AS node
+
+WORKDIR /opt/project
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONPATH .
+ENV TNB_IN_DOCKER true
+ENV TNB_MONGO_HOST mongo
```

Comment:
why this one is needed?

Reply:
I removed just PYTHONPATH. 
PYTHONUNBUFFERED variable avoid buffering of logs. I think it will help to get a latest log in case when container stopped  to work suddenly.
IN_DOCKER - I can be removed. But we decided to leave it. There is another check in if statement. It's os.path.isfile('/.dockerenv').
MONGO_HOST - It's variable to set database host to "mondo" domain name instead of 127.0.0.1. We have to keep 127.0.0.1 ip address for testing purposes and for running dependences like this "make up-dependences-only".